packages/wp-source/types.ts

```diff
@@ -38,6 +38,17 @@ interface WpSource extends Source {
       postsPage: string;
       postEndpoint: string;
       params: Record<string, any>;
+      postTypes: {
+        type: string;
+        endpoint: string;
+        archive?: string;
+      }[];
+      taxonomies: {
+        taxonomy: string;
+        endpoint: string;
+        postTypeEndpoint?: string;
+        params?: Record<string, any>;
```

Comment:
Why have you included this `params`? I don't see it used in the `taxonomyHandler` when you add them.

Reply:
> Why have you included this `params`?

I added that attribute in case you want to use more than one post type for your custom taxonomy; it's for solving the same problem we solve using `state.source.postsEndpoint` and `state.source.params` with posts archives, but for custom taxonomies. 

> I don't see it used in the `taxonomyHandler` when you add them.

Right! I forgot to pass `params` to the `taxonomyHandler` in the `init` action. I'll fix that.

Reply:
Ok. Add a test as well!